src/lib/elements/forms/inputProjectId.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import { FormItem } from '.';
+    import TextCounter from './textCounter.svelte';
+
+    export let value = '';
+    export let autofocus = true;
+
+    let element: HTMLInputElement;
+    let icon = 'info';
+    const pattern = String.raw`^[a-z0-9][a-z0-9-]{1,35}$`;
```

Comment:
Something seems wrong with this pattern when it actually gets built. in the browser console logs i see:

> Pattern attribute value ^[a-z0-9][a-z0-9-]{1,35}$ is not a valid regular expression: Uncaught SyntaxError: Invalid regular expression: /^[a-z0-9][a-z0-9-]{1,35}$/v: Invalid character class

And it doesn't look like the client-side validation works because the network request is sent despite the invalid id:

<img width="1154" alt="image" src="https://github.com/appwrite/console/assets/1477010/95705692-8063-41f0-afbb-ff146734c3ab">